src/detect-csum.c

```diff
@@ -941,7 +941,7 @@ static int DetectCsumValidArgsTestParse01(void)
     FAIL_IF_NULL(de_ctx);\
     Signature *s = DetectEngineAppendSig(de_ctx, "alert ip any any -> any any ("mystr(kwstr)"-csum:vaid; sid:1;)");\
     FAIL_IF(s);\
-    s = DetectEngineAppendSig(de_ctx, "alert ip any any -> any any ("mystr(kwstr)"-csum:invaalid; sid:2;)");\
+    s = DetectEngineAppendSig(de_ctx, "alert ip any any -> any any ("mystr(kwstr)"-csum:invalid; sid:2;)");\
```

Comment:
Puzzled by this one. I didn't take the time to investigate in detail.

Reply:
it appears to be a test to validate a misspelled input is leading to a parsing error